src/envoy/http/mixer/control.cc

```diff
@@ -35,15 +35,19 @@ Control::Control(const Config& config, Upstream::ClusterManager& cm,
                  }) {
   ::istio::control::http::Controller::Options options(config_.config_pb());
 
-  Utils::CreateEnvironment(dispatcher, random, *check_client_factory_,
-                           *report_client_factory_, &options.env);
+  Utils::CreateEnvironment(
+      dispatcher, random, *check_client_factory_, *report_client_factory_,
+      &options.env,
+      config_.config_pb().transport().attributes_for_mixer_proxy());
```

Comment:
will it crash if transport is empty?

Reply:
there is already a use of transport() up above. if it did not crash before, should not crash now.